### DISCUSSION TITLE
Snap Guides in GrapesJs

### DISCUSSION THREAD

**Discussion by phamthanhbinh4796** [GrapesJS/grapesjs#4926] [Q&A]
How can I customize this?
![image](https://user-images.githubusercontent.com/38070904/218700818-1478f961-6896-4764-a5ca-d5180e2624e6.png)

**Comment by artf** [comment]
Here you can find the [source of the command](https://github.com/GrapesJS/grapesjs/blob/f9417578ddeb2d911c5eae2c267442fcbcce67f4/src/commands/view/ComponentDrag.ts) which then you're able to [extend](https://grapesjs.com/docs/modules/Commands.html#extending) as you wish.
